cv_bridge/src/cv_bridge.cpp

```diff
@@ -72,6 +78,22 @@ static int depthStrToInt(const std::string depth) {
 
 int getCvType(const std::string& encoding)
 {
+
+#ifndef USE_BOOST_REGEX
+  std::regex FormatterFormatGroupPattern("\\$\\{([a-zA-Z]+)[}]", std::regex::egrep);
+  // The [}] is a workaround for a bug in at least libstdc++ 5.4 that throws on "\\}" (BUT not on "\\{").
+  using std::match_results;
+  using std::regex_search;
```

Comment:
@HannesSommer Actually do I even need match_results and should I add regex_match?